swagger/src/main/scala/org/http4s/rho/swagger/package.scala

```diff
@@ -25,6 +25,20 @@ package object swagger {
       new PathBuilder(builder.method, PathAST.MetaCons(builder.path, RouteDesc(description)))
   }
 
+  /** Scopes carrier for specific routes */
+  case class RouteSecurityScope(definitions: Map[String, List[String]]) extends SecurityScopesMetaData
+
+  /** Add support for adding documentation before a route using the ## operator */
```

Comment:
I'm not really certain what this is referencing. It mentions `##` and then introduces `^^`.

Reply:
Correct. Typo. Thanks for catching it. I will fix immediately